### DISCUSSION TITLE
Bezier curves can't have their own endpoints?

### DISCUSSION THREAD

**Discussion by Kreijstal** [cetz-package/cetz#138] [Q&A] [answered]
so for example
```typst
cetz.draw.line("pr2.right", "t2out.left", mark: (end: ">"));
```
will show an arrow

but for example()
```typst
cetz.draw.bezier("pr2.right", "t2out.left",(0,0), mark: (end: ">"));
```
won't show any arrow markings.
are there workarounds or am I using this wrong. I want to have arrows that bend. 
Like tikz itself has something similar
```
\draw (t6) edge[bend left=30,->] (p61);
\draw (p61) edge[bend left=30,->] (t0);
```
which are arrows that bend

![image](https://github.com/johannes-wolf/typst-canvas/assets/2415206/08930afb-fb1a-4abf-9d90-5aa72e01dc83)

**Comment by johannes-wolf** [comment]
Added with #140.

This is correct. The only way to add arrows to bezier curves right now is `place-marks`,
a function for placing marks on a path.
It is missing from the docs by accident.

Example:
```
#cetz.canvas({
   import cetz.draw:*
   place-marks(bezier-through((0,0), (1,1), (2,0)),
     (mark: "<", pos:0),
     (mark: ">", pos:1))
})
```
![grafik](https://github.com/johannes-wolf/typst-canvas/assets/519002/d9e7fa79-df43-4bd7-8440-777625e62519)

I will open a ticket/PR for adding `mark:` support to bezier and bezier-through.

**Reply by johannes-wolf** [reply, marked as answer]
Since 0.1.2, they can have marks.

**Reply by Kreijstal** [reply]
I noticed arcs have no marks either, is there a reason why?

**Reply by johannes-wolf** [reply]
With 0.2.0 they have :)
